### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by jakirkham**
Builds the very recent (and long awaited) release of OpenBLAS, 0.3.0. Likely still needs work, but putting it up to get that work and discussion started.

### CODE REVIEW

**Comment by jakirkham** [comment]
Looks like the CIs need some massaging here.

CircleCI is complaining about the build being too quiet. There is a [`no_output_timeout` parameter]( https://circleci.com/docs/2.0/configuration-reference/#run ) which we can set to skirt this. We might consider setting this for all feedstocks ( https://github.com/conda-forge/conda-smithy/issues/784 ).

Travis CI is complaining about the build being too noisy. Most of the warnings are about `fabs` being used when `fabsl` would be more appropriate. We could fix this with a patch or silence the warning. Some discussion about this in issue ( https://github.com/xianyi/OpenBLAS/issues/1357 ).

Surprisingly AppVeyor seems content even after dropping ~10 patches, which is amazing. Good work everyone that has been involved on the Windows build.

**Comment by isuruf** [comment]
> Travis CI is complaining about the build being too noisy.

No, travis-ci is also failing because of timeouts. New openblas versions has fewer warnings and hence no output.

**Comment by jakirkham** [comment]
Yep. You’re right. I misread.

**Comment by jakirkham** [comment]
Would note there is a `.dev` in the SONAME, which is a little odd. ( https://github.com/xianyi/OpenBLAS/issues/1576 )

**Comment by jschueller** [comment]
+1, what's the plan to update all the packages that link to openblas now, old school or conda 3 ?

**Comment by jakirkham** [comment]
Move to `conda-build` 3 as well. We may also take the time to fix `blas` as we rebuild things.

xref: https://github.com/conda-forge/blas-feedstock/pull/12

**Comment by jakirkham** [comment]
Other thoughts?

**Comment by jakirkham** [comment]
Ping?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,40 +7,29 @@ package:
...
+    - perl
+    - flang                      # [win]
+    - cmake                      # [win]
+    - curl                       # [win]
+    - jom                        # [win]
+
+  host:
+    - libflang                   # [win]
+    - libgfortran-ng >=3.0       # [osx]
+    - libgfortran-ng >=4.9       # [linux]
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you remove the two lines above?

Comment (jakirkham) [reply recipe/meta.yaml]: It failed to build without these being present.

Comment (isuruf) [reply recipe/meta.yaml]: Do you have a log?

Comment (jakirkham) [reply recipe/meta.yaml]: Here's one.

ref: https://circleci.com/gh/conda-forge/openblas-feedstock/130

**Comment by jakirkham** [comment]
Anything else?

**Comment by jakirkham** [comment]
Thanks all :)

**Pull request merged**